apps/ledger-live-mobile/src/components/RootNavigator/WalletConnectLiveAppNavigator.tsx

```diff
@@ -46,7 +44,7 @@ export default function WalletConnectLiveAppNavigator() {
 
   const { params: routeParams } = route;
 
-  const platform = walletConnectLiveApp?.params?.liveAppId || "wallet-connect";
+  const platform = "ledger-wallet-connect";
```

Comment:
As it's a const can you move it outside the component, please?

Reply:
Yes :)